### PR TITLE
fix(@schematics/angular): Make manifest.json correct and consistent

### PR DESCRIPTION
**Pull request by webmaxru**
To fix issues mentioned in https://github.com/angular/devkit/issues/775

1) Removed splash_pages from manifest.json
2) Changed display to fullscreen in manifest.json
2) Updated theme-color in index.html
3) Removed Description metatag from index.html (as it doesn't correlate with PWA)

### CODE REVIEW

File: packages/angular/pwa/pwa/files/assets/manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "short_name": "<%= title %>",
   "theme_color": "#1976d2",
   "background_color": "#fafafa",
-  "display": "browser",
+  "display": "fullscreen",
```
Comment (Brocco) [inline packages/angular/pwa/pwa/files/assets/manifest.json]: Please leave the display as browser for the default behavior. Changing it to `fullscreen` could be added as an option for this schematic...
`ng add @angular/pwa --dispaly fullscreen`
But the expected behavior for most users right now should remain as browser.

**Comment by webmaxru** [comment]
@Brocco, with `"display": "browser"` the app cannot be "installed". Requred modes are standalone, fullscreen or minimal-ui (Chrome DevTools output error is a bit wrong because it doesn't list the last one). I believe `minimal-ui` is what we want - a trade-off: app is  installable, but with some UI. Will this work?

https://developer.mozilla.org/en-US/docs/Web/Manifest#display
Btw `minimal-ui`s fallback is exactly `browser`

**Comment by webmaxru** [comment]
Commited with "display": "minimal-ui"

File: packages/angular/pwa/pwa/files/assets/manifest.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "short_name": "<%= title %>",
   "theme_color": "#1976d2",
   "background_color": "#fafafa",
-  "display": "browser",
+  "display": "minimal-ui",
```
Comment (Brocco) [inline packages/angular/pwa/pwa/files/assets/manifest.json]: Per conversations, please update the display property to be "standalone"

**Pull request closed without merging**